Buddies/Topics/TopicsVC.swift

```diff
@@ -25,6 +29,7 @@ class TopicsVC: UICollectionViewController, TopicCollectionDelegate {
         } else {
             selectedTopics = selectedTopics.filter({ $0.id != topic.id })
         }
+        user?.favoriteTopics = selectedTopics.map { $0.id }
```

Comment:
Is this what triggers the update with Firebase?

Reply:
Yep, Jake did a killer job.  Modifying the object updates Firestore.